test/serializers/development_serializer_test.rb

```diff
@@ -65,7 +75,14 @@ def serializer_except
   private
 
     def expected_row
-      [980190962, 562391268, "1970-01-01 00:00:00 UTC", "1970-01-01 00:00:00 UTC", nil, nil, nil, nil, nil, nil, "Godfrey Hotel", "in_construction", nil, nil, nil, nil, "505 Washington Street", "Boston", "MA", "02111", nil, nil, 2016, nil, nil, nil, nil, nil, nil, nil, nil, 75, 12, nil, nil, nil, nil, false, false, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, "Metropolitan Area Planning Council", "http://mapc.org", nil, "Boston, MA", nil, "MAPC", "MAPC", "landlord"]
+      [101010, 562391268, "1969-01-01 05:00:00 UTC", "1969-01-01 05:00:00 UTC",
```

Comment:
Separate every 3 digits in the integer portion of a number with underscores(_).<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.